src/Bank/BankLogic/CheckingAccount.cs

```diff
@@ -54,12 +54,10 @@ public bool ChangeAccount(AccountOrigin AccountOrigin)
 
         public int CompareTo(IBankAccount toCompare)
```

Comment:
Why do we need the compare to if any bussines logic is using it?

Reply:
This time I didn't use it, I will save it for later.